libs/langchain/langchain/schema/runnable/base.py

```diff
@@ -270,7 +270,7 @@ def __ror__(
             Callable[[Iterator[Other]], Iterator[Any]],
             Mapping[str, Union[Runnable[Other, Any], Callable[[Other], Any], Any]],
         ],
-    ) -> Runnable[Other, Output]:
+    ) -> RunnableSerializable[Other, Output]:
```

Comment:
Why is this necessary?

Reply:
Because otherwise you don't get the `.configurable_fields` methods on the return value of a `|`